packages/easy-coding-standard/build/target-repository/.github/workflows/fixer_test.yaml

```diff
@@ -0,0 +1,28 @@
+name: Fixer Test
+
+on:
+    pull_request: null
+    push:
+        branches:
+            - main
+
+jobs:
+    fixer_test:
+        runs-on: ubuntu-latest
+
+        steps:
+            -   uses: actions/checkout@v2
+
+            -   uses: shivammathur/setup-php@v2
+                with:
+                    php-version: '7.3'
+                    coverage: none
+
+            -   run: composer require phpunit/phpunit
+                working-directory: ./tests
+
+            -   run: composer require symplify/easy-coding-standard
+                working-directory: ./tests
```

Comment:
This will resuilt in 2 versions of ECS:

* one after `actions/checkout@v2` as exact `git clone https://github.com/symplify/easy-coding-standard-prefixed` in root directory (probably)
* second after `composer require` in `vendor/symplify/easy-coding-standard`

What do you need both version for testing? The autoload rece-condition/collision issue can happen.

Reply:
Ah sorry, looks like I forgot to removed the checkout. I removed it now.